### PR TITLE
add instances of class Drone and printing array

### PR DESCRIPTION
**Pull request by Wh1teL0rd**

### CODE REVIEW

File: src/ua/lviv/iot/algo/part1/lab1/Drone.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
+package ua.lviv.iot.algo.part1.lab1;
+
+import lombok.*;
```
Comment (BogdanDidukh2003) [inline src/ua/lviv/iot/algo/part1/lab1/Drone.java:3]: https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad

File: src/ua/lviv/iot/algo/part1/lab1/Drone.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+@Setter
+public class Drone {
+    private double currentSpeed;
+    private double currentAltitude;
+    private double batteryCapacity;
+    private double currentBatteryLevel;
+    private static Drone instance;
+
+    public static Drone getInstance() {
+        if(instance == null){
```
Comment (BogdanDidukh2003) [inline src/ua/lviv/iot/algo/part1/lab1/Drone.java]:
```suggestion
        if (instance == null) {
```
Reformat your file please

File: src/ua/lviv/iot/algo/part1/lab1/Drone.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        }
+        return instance;
+    }
+
+
+    public void flyAt(double speedMetersPerMinute, double altitude){
+        this.currentSpeed = speedMetersPerMinute;
+        this.currentAltitude = altitude;
+    }
+    public void chargeBattery(double amount){
```
Comment (BogdanDidukh2003) [inline src/ua/lviv/iot/algo/part1/lab1/Drone.java]:
```suggestion

    public void chargeBattery(double amount) {
```
Reformat your file

File: src/ua/lviv/iot/algo/part1/lab1/Drone.java [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        instance2 = getInstance();
+
+        Drone[] drones = new Drone[4];
+
+        drones[0] = firstDrone;
+        drones[1] = secondDrone;
+        drones[2] = instance1;
+        drones[3] = instance2;
+        int i = 0;
+        while (i<drones.length){
```
Comment (BogdanDidukh2003) [inline src/ua/lviv/iot/algo/part1/lab1/Drone.java]: It is recommended to use foreach (`for ( : )`) loop for iterating through collections

File: src/ua/lviv/iot/algo/part1/lab1/Drone.java (hunk 2 above)
Comment (Wh1teL0rd) [reply src/ua/lviv/iot/algo/part1/lab1/Drone.java]: solved

File: src/ua/lviv/iot/algo/part1/lab1/Drone.java (hunk 3 above)
Comment (Wh1teL0rd) [reply src/ua/lviv/iot/algo/part1/lab1/Drone.java]: solved

**Pull request merged**